src/main/java/org/example/model/Response.java

```diff
@@ -1,9 +1,13 @@
 package org.example.model;
 
+import java.util.List;
+
 public class Response {
     private int code;
     private String body;
     private String contentType;
+    private String jspPage;
+    private List<JspAttribute> jspAttributes;
```

Comment:
The title should be the same as in the getters / setters

Reply:
👽

Reply:
Still wrong

Reply:
fixed